tensorboard/components/vz_example_viewer/vz-example-viewer.html

```diff
@@ -101,54 +102,55 @@
 
       .value-pills-holder {
         flex-grow: 1;
+        flex-basis: 100%;
+      }
+
+      iron-autogrow-textarea {
+        --iron-autogrow-textarea: {
+          font-size: 14px;
+          color: #3C4043;
+        }
+      }
+
+      iron-autogrow-textarea.value-same {
+        --iron-autogrow-textarea: {
+          font-size: 14px;
+          color: #5F6368;
+        }
+      }
+
+      iron-autogrow-textarea.value-different {
+        --iron-autogrow-textarea: {
+          font-size: 14px;
+          color: #34A853;
+          font-weight: 500;
+        }
       }
 
       .value-pill {
         text-align: left;
-        border-radius: 18px;
+        border-radius: 4px;
         margin: 4px;
-        text-overflow: ellipsis;
         padding: 4px 8px;
         border: 1px solid #FFFFFF;
         color: #3C4043;
       }
 
       .value-pill:hover {
-        /* border: 1px solid #ffeb3b; */
         background:#f8f9fa;
       }
 
-      .value-pill:focus {
-        background: white;
-        border: 1px solid #ffeb3b;
-        outline: none;
-      }
-
       .value-compare {
         text-align: left;
-        text-overflow: ellipsis;
+        border-radius: 4px;
         padding: 4px 8px;
-        border: none;
-      }
-
-      .value-different {
-        color: #34A853;
-        font-weight: 500;
-      }
-
-      .value-same {
-        color: #5F6368;
-      }
-
-      input[type=number]::-webkit-inner-spin-button,
-      input[type=number]::-webkit-outer-spin-button {
-        -webkit-appearance: none;
-        margin: 0;
+        border: 1px solid #FFFFFF;
       }
 
       .value-pill-stacked {
         width: 100%;
         margin: 4px 4px 4px 0;
+        padding: 2px;
       }
 
       .value-pill-grid {
```

Comment:
We can remove this rule in future now value-pill-grid is not necessary.

Reply:
plan on keeping these rules in the file in case we re-enable the mode switching or move to another display mode by default.